build_run_docker.sh

```diff
@@ -347,58 +333,43 @@ AUTOBENCH_DBT2_TEMPLATE=$(cat ./resources/config_templates/autobench_dbt2.conf)
 AUTOBENCH_SYSBENCH_TEMPLATE=$(cat ./resources/config_templates/autobench_sysbench.conf)
 
 # Doing restart on-failure for the agent upgrade; we return a failure there
-RONDB_DOCKER_COMPOSE_TEMPLATE="
+service-template() {
+    printf "
 
-    <insert-service-name>:
-      image: $RONDB_IMAGE_NAME
-      container_name: <insert-service-name>
-"
+    %s:
+      image: %s
+      container_name: %s
+" "$SERVICE_NAME" "$RONDB_IMAGE_NAME" "$SERVICE_NAME";
+}
 
 VOLUMES_FIELD="
       volumes:"
 
-# We add volumes to the data dir for debugging purposes
 VOLUME_TEMPLATE="
-      - %s:%s"
+      - type: %s
+        source: %s
+        target: %s"
 
 ENV_FIELD="
       environment:"
 
 ENV_VAR_TEMPLATE="
       - %s=%s"
 
+
 COMMAND_TEMPLATE="
       command: %s"
 
-# Bind config.ini to mgmd containers
-BIND_CONFIG_INI_TEMPLATE="$(printf "$VOLUME_TEMPLATE" "$CONFIG_INI_FILEPATH" "$DATA_DIR/config.ini")"
-
-# Bind my.cnf to mysqld containers
-BIND_MY_CNF_TEMPLATE="$(printf "$VOLUME_TEMPLATE" "$MY_CNF_FILEPATH" "$DATA_DIR/my.cnf")"
-
-BIND_DBT2_SINGLE_CONF_TEMPLATE="$(printf "$VOLUME_TEMPLATE" "$DBT2_CONF_SINGLE_FILEPATH" "$BENCH_DIR/dbt2_single/dbt2_run_1.conf")"
-BIND_DBT2_MULTI_CONF_TEMPLATE="$(printf "$VOLUME_TEMPLATE" "$DBT2_CONF_MULTI_FILEPATH" "$BENCH_DIR/dbt2_multi/dbt2_run_1.conf")"
-
-BIND_AUTOBENCH_SYS_SINGLE_TEMPLATE="$(printf "$VOLUME_TEMPLATE" "$AUTOBENCH_SYS_SINGLE_FILEPATH" "$BENCH_DIR/sysbench_single/autobench.conf")"
-BIND_AUTOBENCH_SYS_MULTI_TEMPLATE="$(printf "$VOLUME_TEMPLATE" "$AUTOBENCH_SYS_MULTI_FILEPATH" "$BENCH_DIR/sysbench_multi/autobench.conf")"
-BIND_AUTOBENCH_DBT2_SINGLE_TEMPLATE="$(printf "$VOLUME_TEMPLATE" "$AUTOBENCH_DBT2_SINGLE_FILEPATH" "$BENCH_DIR/dbt2_single/autobench.conf")"
-BIND_AUTOBENCH_DBT2_MULTI_TEMPLATE="$(printf "$VOLUME_TEMPLATE" "$AUTOBENCH_DBT2_MULTI_FILEPATH" "$BENCH_DIR/dbt2_multi/autobench.conf")"
-
-# This is experimental to optimise performance
-RESOURCES_SNIPPET="
-      ulimits:
-        rtprio:
-          soft: 99
-          hard: 99"
-
 #######################
 #######################
 #######################
 
 echo "Filling out templates"
 
-source $SCRIPT_DIR/docker.env
-source $SCRIPT_DIR/misc_configs.env
+# shellcheck source=./docker.env
```

Comment:
You want to keep this here?
Also, what if the script is executed from somewhere else, does it still work?

Reply:
Yes, it's a linter directive, and is needed to not get warnings about missing variable declarations when they are found in the `.env` files. It's inside a comment so won't affect execution.